tyk-vcs/getter.go

```diff
@@ -194,14 +196,25 @@ func fetchAPIDefinitions(fs billy.Filesystem, spec *TykSourceSpec, subdirectoryP
 }
 
 func fetchAPIDefinitionsDirect(fs billy.Filesystem, spec *TykSourceSpec, subdirectoryPath string) ([]objects.DBApiDefinition, error) {
-	defNames := spec.Files
-	defs := make([]objects.DBApiDefinition, len(defNames))
-	for i, defInfo := range defNames {
-		if defInfo.File == "" {
+	classicAPIFiles := []APIInfo{}
+	oasAPIFiles := []APIInfo{}
+
+	for i := range spec.Files {
+		if strings.HasPrefix(spec.Files[i].File, "oas-") {
```

Comment:
We don't have any requirement on file names that need to processed.
Adding such requirement will not work with existing setup user has.